reader/components/header/Header.jsx

```diff
@@ -44,14 +38,13 @@ const Header = () => {
         <Button
           title="Redirect to CORE metadata page"
           color="none"
-          disabled={pdfMetadata.id === null}
+          disabled={metadata.id === null}
           onClick={() => {
             const coreHostname = 'core.ac.uk'
-            const pdfMetadataSuffix = `display/${pdfMetadata.id}`
+            const pdfSuffix = `display/${metadata.id}`
             if (window.location.hostname === coreHostname)
-              window.location.pathname = pdfMetadataSuffix
-            else
-              window.location = `https://${coreHostname}/${pdfMetadataSuffix}`
+              window.location.pathname = pdfSuffix
+            else window.location = `https://${coreHostname}/${pdfSuffix}`
           }}
```

Comment:
This is a side effect function (modifies global state, i.e. `location`). I prefer and recommend keep multiline functions separate from JSX notations when possible.

It might be hard to read for less skilled developers and for people who knows HTML but are not familiar with JavaScript. Also, it's an indication of good tone.